content/faq/app/which-ios-and-android-versions-are-supported/index.it.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Quali versioni di iOS e Android sono supportate?
```

Comment:
People recommend using quotes for special characters: https://stackoverflow.com/a/22235064